docker-compose-3-persona.yml

```diff
@@ -77,6 +77,7 @@ services:
       --alice
       --unsafe-ws-external
       --unsafe-rpc-external
+      --ws-max-connections 512
```

Comment:
The additional `ws-max-connections` argument is not needed in `docker-compose-3-persona.yml` as we do not run automated tests against this file.